lua/entities/marums_hoverball/init.lua

```diff
@@ -73,14 +75,15 @@ function ENT:PhysicsUpdate()
     if not ( self.damping and self.rotdamping ) then return end
     
     phys:SetDamping( self.damping, self.rotdamping )
-    local tr = util.TraceLine( {
-    start = self:GetPos(),
-    endpos = self:GetPos()+Vector(0,0,-hoverdistance*2),
-    filter = function( ent ) if ( ent:GetClass() == "prop_physics" ) then return false end end,
-    mask = detectmask
-    } )
 
-    local distance = self:GetPos():Distance(tr.HitPos)
+    local tr = util.TraceLine{
+        start = self:GetPos(),
+        endpos = self:GetPos() + Vector(0, 0, -hoverdistance * 2 ),
+        filter = function( ent ) if ( ent:GetClass() == "prop_physics" ) then return false end end,
```

Comment:
Man I don't like how this section is formatted at all. I think `endpos` and `filter` should be defined above `tr` in a better way. Endpos could be broken into two steps to define what the Vector is. Filter should be split up into multiple lines.

Reply:
I agree. im not even sure the filter does anything.

Reply:
Lol, of course it doesn't... 🤦‍♂